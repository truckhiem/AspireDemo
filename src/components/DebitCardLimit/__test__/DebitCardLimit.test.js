import React from 'react';
import {shallow} from 'enzyme';
import {DebitCardLimit} from '../index';

describe('DebitCardLimit', () => {
  it('should render DebitCardLimit', () => {
    const wrapper = shallow(<DebitCardLimit value={100} total={200} />);
    expect(wrapper).toMatchSnapshot();
  });
});
