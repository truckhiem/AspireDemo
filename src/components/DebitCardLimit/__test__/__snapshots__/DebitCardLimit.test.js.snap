// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DebitCardLimit should render DebitCardLimit 1`] = `
<View
  style={
    Object {
      "marginTop": 26,
      "paddingHorizontal": 24,
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "space-between",
        "marginBottom": 6,
      }
    }
  >
    <AspireText
      color="#222222"
      size={13}
      text="Debit card spending limit"
    />
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <AspireText
        color="#01D167"
        size={13}
        text="$100"
      />
      <AspireText
        color="#22222233"
        size={13}
        text="|"
        textStyle={
          Object {
            "marginHorizontal": 5,
          }
        }
      />
      <AspireText
        color="#22222233"
        size={13}
        text="$200"
      />
    </View>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "#01D1671A",
        "borderRadius": 7.5,
        "height": 15,
        "overflow": "hidden",
      }
    }
  >
    <Svg
      height="100%"
      onLayout={[Function]}
      preserveAspectRatio="xMidYMid meet"
      width="100%"
    >
      <Polygon
        fill="#01D167"
        points="0,0 0,0 0,15 0,15"
      />
    </Svg>
  </View>
</View>
`;
